test/stdlib/UnsafeRawBufferPointer.swift

```diff
@@ -407,6 +407,18 @@ UnsafeRawBufferPointerTestSuite.test("subscript.range.wide") {
   buffer[0..<2] = buffer[0..<3]
 }
 
+UnsafeRawBufferPointerTestSuite.test("_copyContents") {
+  let a = Array<UInt8>(0..<20)
+  let b = UnsafeMutableBufferPointer<UInt8>.allocate(capacity: 10*a.count)
+  defer { b.deallocate() }
+  var (unwritten, written) = a.withUnsafeBytes {
+    bytes in
+    bytes._copyContents(initializing: b)
```

Comment:
Out of curiosity, does this test behave any differently before and after your change?

Reply:
The test itself behaves the same, but it runs faster. I wanted to make sure the new code path was explicitly used somewhere in the tests.

Reply:
You could make a microbenchmark out of the original bug and add it to swift/benchmarks if you're so inclined.

Reply:
That's done: see "RawBuffer.copyContents" above.